package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "md-form-builder",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
@kaweesi may be 0.9.0? @rcrichton  published a minor change an hour ago (currently 0.8.0)